routes/users-api.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * All routes for User Data are defined here
+ * Since this file is loaded in server.js into api/users,
+ *   these routes are mounted onto /api/users
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const express = require('express');
```

Comment:
Check the single vs. double quotes usage.  I'd say we choose one to show consistency to the students.  Maybe learning experience managers can weigh in here.

Reply:
You're correct. Regardless of choice, we should be consistent.

Our style guide (https://web.compass.lighthouselabs.ca/activities/801) doesn't state a preference for strings (single vs double), but ours is based on AirBnB's (https://github.com/airbnb/javascript#strings--quotes) and they prefer single. That'd be my preference as well.

Reply:
:+1: -- as long as we're consistent.